packages/docutils/lib/cli/command/init.ts

```diff
@@ -144,17 +157,21 @@ const opts = Object.freeze({
   typescript: {
     default: true,
     description: 'Create tsconfig.json if needed',
-    group: NAME_GROUP_INIT_BEHAVIOR,
+    group: InitCommandGroup.Behavior,
     type: 'boolean',
   },
-}) satisfies Record<string, Options>;
+} as const satisfies Record<string, Options>;
 
 type InitOptions = InferredOptionTypes<typeof opts>;
 
 export default {
   command: 'init',
   describe: 'Initialize package for doc generation',
-  builder: opts,
+  builder(yargs) {
+    return yargs
+      .options(opts)
+      .check(async (argv) => checkMissingPaths(opts, InitCommandGroup.Paths, argv));
```

Comment:
do we need await in the lambda body?

Reply:
we don't have lint support for typescript installed yet (see #18159) so idk.  I was never a fan of that rule